docs/en/configuration.rst

```diff
@@ -186,6 +186,22 @@ file:
 
     export PHINX_ENVIRONMENT=dev-`whoami`-`hostname`
 
+Migration Table
+---------------
+
+To keep track of the migration statuses for an environment, phinx creates
+a table to store this information. You can customize where this table
+is created by configuring ``default_migration_table``:
+
+.. code-block:: yaml
+
+    environment:
+        default_migration_table: phinxlog
+
+If this field is omitted, then it will default to ``phinxlog``. For adapters
+that support it (e.g. Postgres), you can prefix the value with a schema if
+you to have the table in a different schema as the rest of the environment,
```

Comment:
>if you to have the table in a different schema as the rest of the environment

This isn't strictly when you would have to specify the schema. The schema needs to be specified if it's not the default `public` schema. However, I think this whole clause can just be removed because people wanting to specify a schema should already know why they want to do this. I might write this as the following,

_For databases that support it, e.g. Postgres, the schema name can be prefixed with a period separator (`.`). For example, `phinx.log` will create the table `log` in the `phinx` schema._

Reply:
Makes sense. Changed to your suggestion.